### ISSUE TITLE
CoL has no species for the genus Arrenurus Dugès, 1834

### ISSUE THREAD

**Issue by ManonGros** [CatalogueOfLife/data#366]
**Describe the problem**:
While checking this issue on the GBIF feedback portal: https://github.com/gbif/portal-feedback/issues/3797, I realised that there is no species for the genus Arrenurus Dugès, 1834 in the CoL.

I am a bit surprised since ITIS has a number of species for this genus: https://www.itis.gov/servlet/SingleRpt/SingleRpt?search_topic=TSN&search_value=82864#null. But perhaps this is because they are not verified?

(GBIF has 396 species, although there could be many errors: https://www.gbif.org/species/search?rank=SPECIES&highertaxon_key=2181035&status=ACCEPTED)

**Link to effected CoL webpages**:
https://www.catalogueoflife.org/data/taxon/337B

**Literature references**:
I found several publications for various regions but no comprehensive global checklist for this genus (nor water mites in general)

**Comment by yroskov** [comment]
CoL has no provider for the global checklist of Arrenurus. ITIS has an incomplete (regional?) checklist.

**Comment by DaveNicolson** [comment]
Yes, the ITIS data is unverified from NODC, geographic coverage of what they provided is unknown, but likely regional. And the Arrenuridae (and the superfamily Arrenuroidea) are known gaps without an identified global source so far (per the mite gaps spreadsheet I shared with Ed & the taxonomy group, now called MiteHierarchy_Sources/Gaps_2021-10-29_WK).

**Comment by mdoering** [comment]
If ITIS is regional I would also consider other regional sources. The Europeans ones appear to cover more species:

FauEu has 136 species: https://data.catalogueoflife.org/dataset/2026/taxon/174060
EUNIS 142: https://data.catalogueoflife.org/dataset/2122/taxon/x4N7A

**Comment by ManonGros** [comment]
Based on this comment https://github.com/gbif/portal-feedback/issues/3797#issuecomment-992157073 from @aallspac, it seems like a good source for water mites would be the printed book of Viets, K.O. (1987) (see 
[Viets_1987_part_Arrenurus_curtipalpis.pdf](https://github.com/CatalogueOfLife/data/files/7704514/Viets_1987_part_Arrenurus_curtipalpis.pdf)).

Given that the Viets collection is at Senckenberg, perhaps they could help improve the water mite taxonomy in the CoL?

**Comment by yroskov** [comment]
@dhobern, is it for Taxonomy Group?
